### ISSUE TITLE
F-Droid package isn't allowed to access "accessibility features" API on Android 13

### ISSUE THREAD

**Issue by Starbase12** [bitwarden/android#2530]
### Steps To Reproduce

1. Add Bitwardens F-Droid repository to F-Droid
2. Install Bitwarden
3. Login to Bitwarden and access the settings
4. Try to register Bitwarden as accessibility feature

### Expected Result

Be allowed to add Bitwarden as accessibility feature. This should be allowed from apps installed from 3rd party app stores.

### Actual Result

Warning message from Android (Restricted setting)

### Screenshots or Videos

(German news board)
https://m.winfuture.de/news/129489

### Additional Context

Installing the PlayStore version fixed that behavior.

### Operating System

Android

### Operating System Version

13

### Device

Google Pixel 5

### Build Version

latest available 2023-05-21

### Beta

- [ ] Using a pre-release version of the application.

**Comment by ippocratis** [comment]
Apps installed outside google store have accesibility disabled by default

From the system >apps > bitwarden > from that app info page,  on the top right corner , tap on the 3 dots and tap on "allow restricted settings" now you can enable accessibility for bitwarden

**Comment by vvolkgang** [comment]
Issue migrated to https://github.com/bitwarden/mobile/issues/2530
